docs/crictl.md

```diff
@@ -30,12 +30,45 @@ debug: true
 ```
 
 ## Download and Inspect a Container Image
+The pull command tells the container runtime to load a container image from a
+container registry.
 ```console
 $ crictl pull busybox
   ...
 $ crictl inspecti busybox
   ... displays information about the image.
 ```
+
+## Directly Load a Container Image
+Another way to load an image into the container runtime is with the load
+command. With the load command you inject a container image into the container
+runtime from a file. First you need to create the container image. For example
+to create a pause container using Docker:
+```console
+$ docker pull k8s.gcr.io/pause-amd64:3.1
+  3.1: Pulling from pause-amd64
+  67ddbfb20a22: Pull complete
+  Digest: sha256:59eec8837a4d942cc19a52b8c09ea75121acc38114a2c68b98983ce9356b8610
+  Status: Downloaded newer image for k8s.gcr.io/pause-amd64:3.1
+$ docker save k8s.gcr.io/pause-amd64:3.1 > pause.tar
```

Comment:
nit: `-o`

Reply:
yeah works both ways.. docker documents it both ways..

done